### PR TITLE
Upgrade react-native version

### PR DESCRIPTION
**Pull request by tbergq**
Yellow warnings are back 🙈

### CODE REVIEW

File: ios/reactNativeApp.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tbergq) [inline ios/reactNativeApp.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist]: new file from xcode v9.3, shall be committed
[Stack overflow](https://stackoverflow.com/questions/49564513/new-file-created-in-xcode-9-3-wsname-xcworkspace-xcshareddata-ideworkspaceche)

**Comment by mrtnzlml** [comment]
> Yellow warnings are back 🙈

Let's postpone it for a week or so.

**Comment by tbergq** [comment]
Upgraded to 55.1, shipped with correct react version this time, Still yellow warning 🙈

**Comment by mrtnzlml** [comment]
> Upgraded to 55.1, shipped with correct react version this time, Still yellow warning 🙈

I am fine with that. Most of these warnings are from React Navigation and Redux. It should be auto-fixed with future dependency upgrades since there is a big pressure from the community.

We can also disable these deprecated warnings for now (I will probably do it in next PR).

File: app/hotels/src/allHotels/__tests__/__snapshots__/AllHotelsSearchList.test.js.snap [hunk 2]
```diff
@@ -13,7 +13,7 @@ exports[`renders a "not found" message when no hotel is found 1`] = `
 `;
 
 exports[`renders found hotels 1`] = `
-<View>
+<Component>
```
Comment (mrtnzlml) [inline app/hotels/src/allHotels/__tests__/__snapshots__/AllHotelsSearchList.test.js.snap]: @tbergq Do you know what is behind this change?

https://github.com/react-native-community/react-native-releases/blob/master/CHANGELOG.md#055

File: ios/reactNativeApp.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -1176,17 +1177,7 @@
...
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/EvilIcons.ttf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/Feather.ttf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/FontAwesome.ttf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/Foundation.ttf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/Ionicons.ttf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/MaterialCommunityIcons.ttf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/MaterialIcons.ttf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/Octicons.ttf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/SimpleLineIcons.ttf",
-				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/Zocial.ttf",
```
Comment (mrtnzlml) [inline ios/reactNativeApp.xcodeproj/project.pbxproj]: @tbergq My Xcode returned it back. What am I doing wrong?

**Comment by mrtnzlml** [comment]
Integrated, thanks.

**Pull request closed without merging**